### PR TITLE
fix(components): [loading] prevent click through, closes #5646

### LINKED ISSUE #5646: [Bug Report] 同时存在v-loading与@click, 即使在v-loading=true时, click依然会被触发
### Element Plus version
1.3.0-beta.9

### OS/Browsers version
Edge

### Vue version
3.2.29

### Reproduction Link
https://codepen.io/cn-troy/pen/ExbxrXg

### Steps to reproduce
v-loading设置为true, click事件依然可以触发

### What is Expected?
v-loading设置为true, click事件依然可以触发

### What is actually happening?
loading锁定时, click事件应该无效..

### PR DESCRIPTION
**Pull request by weidehai**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.
fix #5646 bug

### CODE REVIEW

File: packages/theme-chalk/src/loading.scss [hunk 1]
```diff
@@ -16,6 +16,11 @@
   @include m(hidden) {
     overflow: hidden !important;
   }
+
+  @include m(prevent) {
+    pointer-events: none;
+    user-select: none;
+  }
```
Comment (msidolphin) [inline packages/theme-chalk/src/loading.scss:23]: I think you can add to the .el-loading-parent--relative

**Review by xiaoxian521 — APPROVED** [review]

**Comment by mikulove666** [comment]
好像有新bug

**Comment by mikulove666** [comment]
新bug就是loading调用close方法后无法点击页面，我看了一下是绑定在遮罩层元素上的类的pointer-events:none这个CSS属性影响了。

**Comment by mikulove666** [comment]
希望作者大大尽快修复这个bug

**Comment by sihuanian** [comment]
> 新bug就是loading调用close方法后无法点击页面，我看了一下是绑定在遮罩层元素上的类的pointer-events:none这个CSS属性影响了。

这个bug已经修复了，issue已经关闭了[https://github.com/element-plus/element-plus/issues/9662](url)

**Comment by languanghao** [comment]
终于找到了，突然发现ElMessage点不了了，原来是这个造成的

**Pull request merged**